Libraries/zephyr/MAX/Source/MAX32657/CMakeLists.txt

```diff
@@ -22,6 +22,12 @@ if(NOT TARGET_REV)
   zephyr_compile_definitions(-DTARGET_REV=0x4131)
 endif()
 
+if(CONFIG_TRUSTED_EXECUTION_SECURE)
+  zephyr_compile_definitions(-DIS_SECURE_ENVIRONMENT=1)
```

Comment:
Not essential, just a suggestion,
How about to align msdk and zepyhr flags,
On zephyr side CONFIG_TRUSTED_EXECUTION_SECURE" and CONFIG_TRUSTED_EXECUTION_NONSECURE is used to generate secure, non-secure image.

If you replace IS_SECURE_ENVIRONMENT with CONFIG_TRUSTED_EXECUTION_SECURE in msdk files. MSDK will be align with zephyr.

Reply:
Since many members of the roadtest and verification teams are used to the current flags, I'll still leave `IS_SECURE_ENVIRONMENT` defined, but I'll use the `CONFIG_TRUSTED_EXECUTION_*` going forward.

Reply:
Thanks, in that case I can remove conversion in Libraries/zephyr/MAX/Source/MAX32657/CMakeLists.txt.
Let me I apply some test with final status of this branch on zephyr side.

Reply:
It's removed.